draft-demarco-oauth-status-assertions.md

```diff
@@ -355,7 +355,7 @@ encoding:
 .
 {
     "iss": "0b434530-e151-4c40-98b7-74c75a5ef760",
-    "aud": "https://issuer.example.org/status-assertion-endpoint",
+    "aud": "https://issuer.example.org/status",
```

Comment:
`status` alone is not auto explicative as I'd liek to have in this non-normative example

status alone is too much generic